rzup/src/cmd/extension.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2024 RISC Zero, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use clap::Subcommand;
+
+use crate::{
+    extension::{install::InstallExtension, repo::ExtensionRepo},
+    help,
+};
+
+#[derive(Debug, Subcommand, Clone)]
+pub enum ExtensionSubcmd {
+    /// Use a specific cargo-risczero version
+    #[command(aliases = ["install", "add"])]
+    Use {
+        #[arg(help = help::INSTALL_HELP)]
+        version: String,
+    },
+    /// Update cargo-risczero extension
+    Update,
```

Comment:
Can these just be one command? Confusing that they do the same (as far as I can see) but one just handles installs a versioned?

Reply:
`use` is for using specific versions of `cargo-risczero`. 
`update` checks for all uneatable items (tool chains and extension) and updates, if any are available. Definitely confusing thanks for pointing it out!